app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,11 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
+      if(exportData !== undefined && exportData.Keystore !== undefined && exportData.Keystore.length > 0) {
+        Alerting.AlertAddMsg('bottom', 'warning', 'the key will be exported and should be protected');
```

Comment:
Maybe make the description more detailed: 'Warning: The server's TLS private key will be exported and should be protected!'